src/main/java/io/devlabs/keytree/domains/company/domain/Admin.java

```diff
@@ -0,0 +1,41 @@
+package io.devlabs.keytree.domains.company.domain;
+
+import io.devlabs.keytree.commons.jpa.BaseTimeEntity;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import org.hibernate.annotations.Comment;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Getter
+@Builder
+public class Admin extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Comment("기업 계정 고유 아이디")
+    private Long id;
+
+    @Column
+    @NotNull
+    @Comment("계정 이름")
+    private String name;
+
+    @Column
+    @NotNull
+    @Comment("로그인 아이디")
+    private String loginId;
+
+    @Column
+    @NotNull
+    @Comment("로그인 비밀번호")
+    private String password;
+
+    @Column
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    @Comment("권한")
+    private UserRole authority;
+}
```

Comment:
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Comment("관리자 아이디")
    private Long id;

    @Column
    @NotNull
    @Comment("이름")
    private String name;

    @Column
    @NotNull
    @Comment("이메일")
    private String email;

    @Column
    @NotNull
    @Comment("비밀번호")
    private String password;

    @Column
    @NotNull
    @Enumerated(EnumType.STRING)
    @Comment("관리자 권한")
    private UserRole userRole;
}
```
- 아이디 정책이 현재 메일이라 email로 사용하는 편이 나을것 같습니다.
- 이외 코멘트 변경, 역할 관련 필드명 변경 있습니다.